euchre/data_structures/card.py

```diff
@@ -1,13 +1,31 @@
 class Card:
 
-    """
-    - features
-		○ Suit
-		○ Value
-	- Functions
-		○ Get_color
-		○ Is_playable(other_card) - do we need this? I think so
-		○ Compare_whos_best(card)
-    """
-    def __init__(self):
-        raise NotImplementedError
+    def __init__(self, rank, suit):  # initializes card with rank and suit attributes
+        self.rank = rank
+        self.suit = suit
+
+    def Get_rank(self):  # gets the rank
+        return self.rank
+
+    def Get_suit(self):  # gets the suit
+        return self.suit
+
+    def Get_color(self):  # gets the color
+        if self.Get_suit() == 'Hearts' or self.Get_suit() == 'Diamonds':
+            return 'Red'
+        else:
+            return 'Black'
+
+    def Is_playable(self):
+        return None  # should return True if card either has same suit as first card played or no card in hand matches
+        # the suit of first card played.  Should be a function of Hand?
+
+    def Compare_whos_best(self,
```

Comment:
I dunno, maybe this should be somewhere else

Reply:
Let's make Conner do it.